README.md

```diff
@@ -31,13 +31,13 @@ POST http://localhost:8080/api/put
 Content-Type: application/json
 
 {
```

Comment:
Ok so here we need to mention that we can use a list also. That really threw me off...